CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+# Contributing Guidelines
+---
+
+
+Firstly thanks for your contributions!!! :sparkling_heart::sparkling_heart:
+
+If you have resources you would like to share from your event or resources you think our community
+would benefit from keep reading!
+
+## PRE-REQUISITE
+
+1. 📖 Read up on fork & pull request models
+2. 🍴 Fork this repo to your account
+3. 🌱 Create a branch for the change you intend to make in your fork
+4. ✍️ Make your changes to the above created branch in your fork
+5. 🔨 Follow the contributing guidelines below
+6. 🔧 Send a pull request from your fork's branch to our `master` branch
```

Comment:
optional - instead of "master" we can mention as 'main'

Reply:
Yes!